### PR TITLE
How it works

### PR DESCRIPTION
**Pull request by jmoney618**
I think I finished the 'How It Works' section.  It's not responsive right now though

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -1,16 +1,21 @@
 body {
```
Comment (samanthaberk) [inline styles/main.css:1]: I think adding some space between the sections would make it more reader-friendly. Maybe we could make a new style rule for `section` and add `margin-top: 60px;`

File: styles/main.css [hunk 2]
```diff
@@ -24,25 +29,74 @@ body {
...
+}
+
+.card {
+  align-items: center;
+  background-color: #40596b;
+  border-radius: 5px;
+  margin: 1rem;
+  position: relative;
+  min-height: 30rem;
+}
```
Comment (samanthaberk) [inline styles/main.css]: capture
I thought adding a shadow effect around each of the card divs might look cool. Here is how I did it in Chrome Dev Tools:

`box-shadow: 5px 4px 3px rgba(64,89,107, 0.5), 0px 8px 13px rgba(0,0,0,0.4), 0px 18px 23px rgba(0,0,0,0.4);`

This is just cosmetic and I know not everyone likes this kind of shadow effect, so if you like the clean look better that's totally okay with me!

**Review by samanthaberk — COMMENTED** [review]
First check done. Looks really great!

**Comment by jmoney618** [comment]
I added the box shadow Sam suggested to the card class

**Pull request merged**